tests/backend/test_ansible_installation.py

```diff
@@ -456,3 +457,26 @@ async def test_host_key_verification_with_ssh_options():
             known_hosts.unlink()
         if backup_path and backup_path.exists():
             backup_path.rename(known_hosts)
+
+@pytest.mark.asyncio
+async def test_host_location_given_hostID():
```

Comment:
Lets not put the hostID like that instead use host_id.  I think that's in more than this place, but look over the code and make sure.